doc/usecases/polyculture/search_plants.md

```diff
@@ -0,0 +1,29 @@
+# Use Case: Search Plants
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can search for plants to add to her map and sort results by different criterion.
+- **Status:** Draft
```

Comment:
This can be assigned to Moritz and Paul. (like #115  doc/usecases/map_search.md), they actually already work on it.

@badnames @Bushuo